lib/rules/font-family-no-missing-generic-family-keyword/index.js

```diff
@@ -48,6 +48,15 @@ function rule(actual) {
 				return;
 			}
 
+			if (
```

Comment:
We'll want to use the `optionsMatches` util here, [like so](https://github.com/stylelint/stylelint/blob/8c25249173dedd93f72c8b4d966abf67cb54f1ad/lib/rules/property-no-unknown/index.js#L64). So that we support regex as well as strings.

Reply:
Cool, that makes sense. Simplifies my work a bit